Example/ViewController+FreeDrive.swift

```diff
@@ -26,6 +26,20 @@ extension ViewController {
         subscribeForFreeDriveNotifications()
     }
     
+    func setupCongestionMapping() {
+        let congestionMapping: CongestionMapping = { (numericLevel, roadClass) in
+            switch roadClass {
+            case .motorway, .primary:
+                CongestionRange.setCongestionRanges(low: 0..<40, moderate: 40..<60, heavy: 60..<80, severe: 80..<101)
```

Comment:
`CongestionRange.setCongestionRanges` will only affect the "default" congestion ranges. This implementation will switch them for every road class and only the values set in the last iteration will make an effect.